rdm/checklists/FDA-HFE_2011.txt

```diff
@@ -15,5 +15,10 @@ FDA-HFE:4.2.c Process for human factors validation testing
 FDA-HFE:5.1 Document device users
 FDA-HFE:5.2 Document device use environment
 FDA-HFE:5.3 Document device user interface
+FDA-HFE:5.4 Identify all user tasks and their associated risks
+FDA-HFE:5.6 Specify the user interface design
+FDA-HFE:5.7 Establish a user interface evaluation plan
+FDA-HFE:5.7 Specify wether medical device specific training is required
+FDA-HFE:5.7 Specify wether accompanying documentation is provided
```

Comment:
Nice job updating the FDA-HFE checklist! Note that the identifier on each line will need to be unique so that we can trace from these lines to the documents that fulfill them. Thus, instead of `FDA-HFE:5.7` you may need `FDA-HFE:5.7.a` and `FDA-HFE:5.7.b`, or something similar.